teams/teams-ps/teams/New-CsTeamTemplate.md

```diff
@@ -645,158 +639,162 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ### Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.IConfigApiBasedCmdletsIdentity
 
-### Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ICreateTemplateFromRequest
+### Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ITeamTemplate
 
 ## OUTPUTS
 
 ### Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ICreateTemplateResponse
 
-### Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.IErrorObject
+### Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ITeamTemplateErrorResponse
 
 ## NOTES
 
-COMPLEX PARAMETER PROPERTIES\

```

Comment:
I think I have originally misformatted the complex parameter properties. I have not manually tweaked them, it's just the visual changes